src/options.js

```diff
@@ -616,12 +616,18 @@ export function maybeNamed(things) {
   return isIterable(things) ? named(things) : things;
 }
 
+// A shared Sphere object coalesces all sphere clips.
+const sphere = {type: "Sphere"};
```

Comment:
This is for better memoization later?

Reply:
yes — we don't want each `"sphere"` and `{type:"Sphere"}` to generate a new clip-path